templates/package/partials/favorites.html

```diff
@@ -15,11 +15,12 @@
         <span class="glyphicon glyphicon-heart"></span>
         Favorited
     </div>
-    <dialog>
-        <h3>
+
+    <dialog class="tw:backdrop-brightness-50 tw:backdrop-blur-sm tw:z-10 tw:mt-[100px] tw:border-none tw:rounded-2xl tw:bg-white tw:shadow-lg   tw:p-4 tw:mx-auto tw:mt-20" id="dialog">
+        <h3 class="tw:text-lg tw:font-semibold tw:text-center">
             Do you want to unfavorite {{ package.title }}?
         </h3>
-        <div class="action ">
+        <div class="tw:flex tw:justify-end tw:mt-4 tw:gap-4">
```

Comment:
👍 porting the existing style tag to tailwind

Just need to clean up the extra space before the dialog element.